Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim
```

Comment:
This should be 3.9.12 for Wander to have it working too on M1 Mac

Reply:
But this is just the base of the docker image. @wandersiemers doesn't any type of container work on your mac?

Reply:
3.9 doesn't have ARM support, 3.9.12 does, so no, 3.9 base image containers don't work on my machine

Reply:
There is no 3.9.12 image on dockerhub, but I assume a 3.9.13 image would work fine?

Reply:
Seems to work just fine? 
<img width="629" alt="image" src="https://user-images.githubusercontent.com/24796206/171741878-1b96e08e-20e1-452a-8b19-c4513e51d188.png">

Reply:
![Screenshot 2022-06-06 102459](https://user-images.githubusercontent.com/33878402/172124343-956aafa5-6e75-489b-b9cc-c4278e6aa2fb.png)
Right, I was looking here on dockerhub and didn't see it

Reply:
Changed it